app/subscriber/src/features/my-folders/ConfigureFolder.tsx

```diff
@@ -108,47 +107,46 @@ export const ConfigureFolder: React.FC<IConfigureFolderProps> = ({ active }) =>
             ],
             (item) => item.content.id,
           ).map((item, index) => ({ ...item, sortOrder: index }));
-          await updateFolder({ ...currentFolder, content });
+          await updateFolder({ ...active, content });
           await findMyFolders();
           toast.success(`Filter found and added ${results.hits.hits.length} content items.`);
         } else {
           toast.warning('No content found for this filter.');
         }
       } catch {}
     },
-    [findContentWithElasticsearch, currentFolder, updateFolder, findMyFolders],
+    [findContentWithElasticsearch, active, updateFolder, findMyFolders],
   );
 
   const handleSaveFolder = React.useCallback(
     async (values: IFolderModel) => {
       try {
         const result = await updateFolder(values);
-        setCurrentFolder(result);
+        setActiveFolder(result);
+        setActiveFolder(result);
         toast.success('Folder schedule saved.');
       } catch {
         toast.error('Failed to save folder schedule.');
       }
     },
-    [updateFolder, setCurrentFolder],
+    [updateFolder, setActiveFolder],
   );
 
   return (
     <styled.ConfigureFolder
       ignoreLastChildGap
       header={
         <Row width={FieldSize.Stretch}>
-          <FaGear className="gear" /> {currentFolder?.name}
+          <FaGear className="gear" /> {active?.name}
         </Row>
       }
     >
       <div className="main-container">
         <Text
           name="name"
           label="Folder name:"
-          value={currentFolder?.name ?? ''}
-          onChange={(e) =>
-            setCurrentFolder({ ...currentFolder, name: e.target.value } as IFolderModel)
-          }
+          value={active?.name ?? ''}
+          onChange={(e) => setActiveFolder({ ...active, name: e.target.value } as IFolderModel)}
         />
         <h2>Folder automation settings</h2>
         <Row>
```

Comment:
@jdtoombs The checkbox on line 161 doesn't do anything.  It needs to be hooked up to enabling the schedule.